### PR TITLE
Add templates for GitHub issues

### PR DESCRIPTION
**Pull request by beeb**
The templates cover Bug Reports and Feature Requests. More can be added later.

If we start using tags to triage issues, those can be automatically assigned by changing the frontmatter of these new files.

When appropriate, we can add the following for instance:

```yml
labels: kind/bug, status/triage
---
```

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+name: "\U0001F381 Feature Request"
```
Comment (beeb) [inline .github/ISSUE_TEMPLATE/feature_request.md:2]: "Gift" emoji

**Comment by nvarner** [comment]
Is it possible to use 🐞 and 🎁 directly, rather than their Unicode codepoints?

**Comment by beeb** [comment]
Sure, I'll change it in a few minutes

**Comment by nvarner** [comment]
Looks good. Thank you!

**Pull request merged**